### DISCUSSION TITLE
[Feature] AI photo enhancement

### DISCUSSION THREAD

**Discussion by arne182** [immich-app/immich#14904] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Either automatically generated or on demand running the AI upscaler/enhancer

https://www.topazlabs.com/topaz-photo-ai

A similar one is also available for video. 

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#1795
